crud/tarantool_test.go

```diff
@@ -40,10 +40,62 @@ var startOpts test_helpers.StartOpts = test_helpers.StartOpts{
 var timeout = float64(1.1)
 
 var operations = []crud.Operation{
+	// Insert new fields,
+	// because double update of the same field results in an error.
 	{
-		Operator: crud.Assign,
```

Comment:
We don't test assign now? It's the most useful update operation.

Reply:
Oops, restored the assign test.